src/CSTParser.jl

```diff
@@ -59,7 +59,15 @@ function parse_expression(ps::ParseState, esc_on_error = false)
             if both_symbol_and_op(ps.t)
                 ret = EXPR(:IDENTIFIER, ps)
             else
-                ret = INSTANCE(ps)
+                if ps.t.dotop && closer(ps) && !isassignmentop(ps.t)
+                    # Split dotted operator into dot-call
+                    v = val(ps.t, ps)[2:end]
+                    dot = EXPR(:OPERATOR, 1, 1, ".")
+                    op = EXPR(:OPERATOR, ps.nt.startbyte - ps.t.startbyte - 1, ps.t.endbyte - ps.t.startbyte, v)
+                    ret = EXPR(dot, EXPR[op], nothing)
+                else
+                    ret = INSTANCE(ps)
+                end
```

Comment:
I think this all needs to be wrapped in a `@static if VERSION < v"1.6" ... else ret = INSTANCE(ps) end` to match old parsing behaviour. Not clear if v1.6 is the correct cut off.